src/app/shared/interfaces/report.ts

```diff
@@ -7,13 +7,15 @@ export interface Report extends BaseReport {
   crudStorage: boolean;
   endTime: number;
   estimatedMemoryUsage: number;
+  // TODO: Add "path"?
   fullPath: string;
   inputCheckpoint: Checkpoint;
   numberOfCheckpoints: number;
   originalEndpointOrAbortpointForCurrentLevel?: Checkpoint;
   originalReport: Report;
   reportFilterMatching: boolean;
   startTime: number;
+  // TODO: Remove, is not provided by the backend.
```

Comment:
Why add this as a todo and not just remove it?

Reply:
I do not want to touch the compare component right now.